emhttp/plugins/dynamix/scripts/manage_boot_params.sh

```
@@ -459,10 +526,16 @@ validate_custom_param() {
         error_exit "Invalid custom parameter: contains spaces. Only one parameter per entry allowed."
     fi
 
-    # Check 3: Reserved syslinux directives (case-insensitive)
+    # Check 3: Reserved directives (case-insensitive)
     local lower_param=$(echo "$param" | tr '[:upper:]' '[:lower:]')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)